### DISCUSSION TITLE
Why i have to set Speeds in TFT Firmware

### DISCUSSION THREAD

**Discussion by asusdk** [bigtreetech/BIGTREETECH-TouchScreenFirmware#1951] [Q&A]
Today i updated the Firmware of my TFT35, since then the Printer moves like a Freak, so fast like Flash, why, should it not take the Settings from the Marlin Firmware or at least from the Gcode File ??

**Comment by kisslorand** [comment]
Give some more details. Moves fast while printing? Moves fast when not printing?
There are settings for the non-printing moves (slow, normal, fast).  You can set the three move spped in `config.ini`.
